packages/core/src/Editor.ts

```diff
@@ -427,14 +427,15 @@ export class Editor extends EventEmitter<EditorEvents> {
     blockSeparator?: string
     textSerializers?: Record<string, TextSerializer>
   }): string {
-    const { blockSeparator = '\n\n', textSerializers = {} } = options || {}
+    const { blockSeparator = '\n\n', textSerializers = {}, repetitiveSeparator = false } = options || {}
```

Comment:
I'm not sure about the name of this option. For me it wouldn't be really clear what is meant by this.

Reply:
Thanks for the comment. Please suggest a name that would be convenient?

Reply:
Not sure exactly. From what I see this is adding the blockSeperator even when blocks are not separated?

Reply:
maybe `forceBlockSeperator`? I'd just like to explain to the users what this option exactly does. Maybe keep the name and add a jsDoc explaining the option?

I myself couldn't exactly tell what the option does just from the name.

Reply:
I'd go for `allowRepetitiveSeparators` to indicate that this allows repetitive separators to be parsed